File: 242.Valid_Anagram.cs
public class Solution
{
    public bool IsAnagram(string s, string t)
    {
        var cnt = new int[123];
        foreach (var x in s) cnt[x]++;
        foreach (var x in t) cnt[x]--;
        foreach (var x in cnt) if (x != 0) return false;
        return true;
    }
}

File: 242.Valid_Anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False
        ls = [0] * 123
        for i in range(len(s)):
            ls[ord(s[i])] += 1
            ls[ord(t[i])] -= 1

        for num in ls:
            if num != 0:
                return False
        
        return True